artix7/tests/picosoc/README.md

```diff
@@ -0,0 +1,8 @@
+# PicoSoC
+
```

Comment:
Can we share PicoSoC between ice40 and artix7?

Reply:
Sure, but for now I'll just remove the projects and add back in another pull request.